babel.config.js

```diff
@@ -1,21 +0,0 @@
-module.exports = {
-    presets: [
-        [
-            '@babel/preset-env',
-            {
-                targets: {
-                    node: 'current',
-                },
-            },
-        ],
-    ],
-    plugins: [
-        [
-            'formatjs',
-            {
-                idInterpolationPattern: '[sha512:contenthash:base64:6]',
-                ast: true,
-            },
-        ],
-    ],
```

Comment:
According to the docs, this is only needed to add `id` property to `FormattedMessage` elements from `react-intl`:

https://formatjs.io/docs/guides/bundler-plugins

So it was not used in this repo at all.